pkg/crc/api/handlers.go

```diff
@@ -2,24 +2,26 @@ package api
 
 import (
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 
 	"github.com/code-ready/crc/cmd/crc/cmd/config"
 	crcConfig "github.com/code-ready/crc/pkg/crc/config"
 	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/errors"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/crc/pkg/crc/machine"
 	"github.com/code-ready/crc/pkg/crc/validation"
 	"github.com/code-ready/crc/pkg/crc/version"
 )
 
-func statusHandler(client machine.Client, _ ArgsType) string {
+func statusHandler(client machine.Client, _ json.RawMessage) string {
 	statusConfig := machine.ClusterStatusConfig{Name: constants.DefaultName}
 	clusterStatus, _ := client.Status(statusConfig)
 	return encodeStructToJSON(clusterStatus)
 }
 
-func stopHandler(client machine.Client, _ ArgsType) string {
+func stopHandler(client machine.Client, _ json.RawMessage) string {
```

Comment:
@anjannath should this be in a different commit message where you changed the `ArgsType` to `json.RawMessage` ?

Reply:
Yeah, makes sense to separate this,will do thanks!